stdlib/3.7/_py_abc.pyi

```diff
@@ -0,0 +1,10 @@
+
+from typing import Type, TypeVar, Tuple, Any, Dict
+
+_T = TypeVar('_T')
+
+def get_cache_token() -> object: ...
+
+class ABCMeta(type):
+    def __new__(mcls, __name: str, __bases: Tuple[Type[Any]], __namespace: Dict[str, object], **kwargs) -> ABCMeta: ...
```

Comment:
```suggestion
    def __new__(mcls, __name: str, __bases: Tuple[Type[Any]], __namespace: Dict[str, Any]) -> ABCMeta: ...
```

* `object` -> `Any`, otherwise passing most dicts would fail due to `Dict`'s invariance(?).
* While the declaration allows `kwargs`, these get passed to `object.__new__()`, which doesn't accept them. I get a `__init_subclass__() takes no keyword arguments` when trying to use `kwargs` in `ABCMeta.__new__()` calls.